### PR TITLE
Cubit 2020 compatibility

### PR DESCRIPTION
**Pull request by bam241**
this is an attempt to update the Plugin for compatibility with Trelis 17.1 and Cubit 2020.2

### CODE REVIEW

File: SvalinnPlugin.cpp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #include <sstream>
 
 #include "CubitMessageHandler.hpp"
-#include "CubitInterface.hpp"
+#include "CubitCoreformInterface.hpp"
```
Comment (gonuke) [inline SvalinnPlugin.cpp]: What if this was something like:
```
#ifdef CUBIT_VERSION < XX.YY
#include "CubitInterface.hpp"
#define MESSAGE_HANDLER CubitInterface::get_cubit_message_handler()
#else
#include "CubitCoreformInterface.hpp"
#define MESSAGE_HANDLER CubitMessage::get_message_handler()
#endif
```
and below is something like:

``` 
CubitMessageHandler* console = MESSAGE_HANDLER;
```

Comment (bam241) [reply SvalinnPlugin.cpp]: I did something similar

but in a header file so it doesn't add to much copy paste...

**Comment by bam241** [comment]
@pshriwise @gonuke this is ready to be reviewed,

**Comment by gonuke** [comment]
This LGTM - do we have any actions or CI testing this?

**Comment by bam241** [comment]
sadly not yet...

**Comment by bam241** [comment]
I could merge it with #78  and then both would be tested

**Comment by gonuke** [comment]
It probably makes sense to merge these - add build changes with testing changes

**Comment by bam241** [comment]
closing there as it has been merged with #78

**Pull request closed without merging**